components/services/bat_ledger/bat_ledger_app.cc

```diff
@@ -27,9 +27,9 @@ void OnBatLedgerServiceRequest(
 }  // namespace
 
 BatLedgerApp::BatLedgerApp(
-        service_manager::mojom::ServiceRequest request) :
-    service_binding_(this, std::move(request)),
-    service_keepalive_(&service_binding_, base::TimeDelta()) {
+    service_manager::mojom::ServiceRequest request) :
```

Comment:
remove this change as we just changed spacing which was fine before. Why I want it in is to remove noise in this PR

Reply:
removed